### ISSUE TITLE
Choose R Window keyboard support incomplete (Electron)

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#11098]
### System details

    RStudio Edition : Desktop (Electron)
    RStudio Version : 2022.06.0-daily+343
    OS Version      : Windows 10 or 11
    R Version       : Any

### Steps to reproduce the problem

- Start RStudio
- Tools / Global Options, click on Change... button to display the Choose R dialog
- Try to use the dialog via keyboard

### Describe the problem in detail

- <kbd>Esc</kbd> key doesn't close the window (should be equivalent to clicking Cancel)
- <kbd>Enter</kbd> key doesn't close the window (Enter should be equivalent to clicking OK)
    - related to that, ideally but much lower priority, the Enter key would have highlighting indicating it is the default button for the dialog; e.g. here's how it looks in the Qt dialog, notice the blue outline, this is separate from the focus indicator:

![qt](https://user-images.githubusercontent.com/10569626/166071017-bf24f7a1-02ae-4073-bc16-38cde82e4f31.png)

### Describe the behavior you expected

- Standard Windows dialog box keyboard behaviour
- Have Esc close dialog is the most important; making the Enter key the default button and having a visual indicator is a nice-to-have

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by ronblum** [comment]
The first part now works. However, after clicking on `Esc`, keyboard navigation no longer works in the Global Options modal until it's clicked on with the mouse. That is

1) Follow the steps above
2) `Esc`
3) `Esc`, `Enter`, `Tab`, etc. don't work.

**Comment by ronblum** [comment]
Verified that this is working in RStudio Desktop 2022.06.0-daily+482 (Electron).

**Comment by ronblum** [comment]
FWIW, the `quarto` package doesn't find the bundled Quarto on Windows, though it works on MacOS 12.5, Red Hat 8.5, and Ubuntu 22.04.

````
> quarto::quarto_path()
NULL
> quarto::quarto_inspect()
Error in find_quarto() : Unable to find quarto command line tools.
````

It's also not on the PATH, checked with `Sys.getenv("PATH")`.
